### ISSUE TITLE
I turned off all antiviruses on my pc and added exeptions just i case.

### ISSUE THREAD

**Issue by RlyJustAnAverageGuy** [AllToMP3/alltomp3-app#145]
When i try to download a playlist from spotify the progress bar doesn't move at all even after an hour of waiting. I tried changing the folder and turning off windows defender but it doesn't seem to do much of anything. Did someone else experience the same or similar problem and if so, how did you solve it?

**Comment by tomquental** [comment]
Yes, I've had the same problem and it seems like this has been going on for multiple months at this point, like this: [Stuck downloading Spotify Playlist #139](https://github.com/AllToMP3/alltomp3-app/issues/139).

Supposedly, a workaround has been written in this [PR](https://github.com/AllToMP3/alltomp3/pull/27). But I (a layman in programming) am unable to understand how to write this into the files and try to run it again.

I'm using windows 10
alltomp3 version 0.3.11.128
